components/brave_wallet_ui/common/slices/api.slice.ts

```diff
@@ -1435,41 +1445,100 @@ export function createWalletApi (
       }),
       getTokenBalancesForChainId: query<
         TokenBalancesForChainId,
-        GetTokenBalancesForChainIdArg
+        GetTokenBalancesForChainIdArg | typeof skipToken
```

Comment:
I don't think you need to define this. All queires should automatically accept `skipToken` as an arg.